task3/update/styles.css

```diff
@@ -0,0 +1,159 @@
+header{
+    position: fixed; 
+    left: 0; top: 0; 
+    padding: 5px; 
+    background: #90EE90; 
+    width: 100%; 
+    height: 60px;
+    z-index: 3;
+}
+.logo{
+    position: fixed;
+    top: 15px;
+}
+.logotext{
+    position: fixed;
+    top:-30px;
+    font-size: 34px;
+    color: #39b54a;
+}
+.write{
+    position: fixed;
+    top: 32px;
+    left: 340px;
+}
+.exit{
+    position: fixed;
+    right: 200px; top:0;
+}
+.user{
+    position: fixed; 
+    right: 0; top: 0;
+}
+.yourname{
+    position: fixed;
+    right: 75px; top: 15px;
+    font-size: 28px;
+    color: black;
+    border-bottom: 2px solid black;
+}
+aside{
+    position: fixed; 
+    width: 20%;
+    left: 0; top: 70px;
+    bottom: 45px;
+    background: white; 
+    z-index: 2;
+    
+}
+.prefs{
+    position: fixed; 
+    width: 20%;
+    left: 1%; top: 28%;
+    font-size: 14px;
+}
+.filter{
+    background: #C0C0C0; 
+    position: fixed;
+    left: 1%; top: 35%;
+    bottom: 22%; right: 81%;
+}
+.typein{
+    width: 80%;
+}
+.day{
+    width: 7%;
+}
+.month{
+    width: 12%;
+}
+.year{
+    width: 15%;
+}
+.filtersubmit{
+    position: fixed;
+    left: 14%; bottom: 76px;
+}
+body{
```

Comment:
css is a code, 
clean code should be structured,
please group and order selectors by the placement and purposes